### PR TITLE
Prepare site.yml and namespace.js for Searchisko 2

### PR DESCRIPTION
**Pull request by lukas-vlcek**
See https://issues.jboss.org/browse/RHD-559 for details.

### CODE REVIEW

**Comment by paulrobinson** [comment]
this is ok to test

File: javascripts/namespace.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-interpolate: true
+    interpolate: true
```
Comment (paulrobinson) [inline javascripts/namespace.js]: Can you remove this 'white-space' change please.

**Comment by paulrobinson** [comment]
Otherwise, this looks fine.

**Comment by lukas-vlcek** [comment]
@paulrobinson I updated the PR code and removed extra white-space. Should be ok now.

**Pull request merged**